logic/account.js

```diff
@@ -404,9 +404,10 @@ class Account {
 		const totalSupplyBignum = new Bignum(totalSupply);
 		const approvalBignum = votersBalanceBignum
 			.dividedBy(totalSupplyBignum)
-			.times(100)
-			.round(2);
-		return !approvalBignum.isNaN() ? approvalBignum.toNumber() : 0;
+			.multipliedBy(100)
+			.toFixed(2);
+
+		return !isNaN(approvalBignum) ? Number(approvalBignum) : 0;
```

Comment:
Then we can go with previous logic here, but I suggest changing `!approvalBignum.isNaN()` to `approvalBignum.isFinite()`